protocols/v2/noise-sv2/src/responder.rs

```diff
@@ -226,9 +248,27 @@ impl Responder {
     ///
     /// On failure, the method returns an error if there is an issue during encryption, decryption,
     /// or any other step of the handshake process.
+    #[cfg(feature = "std")]
     pub fn step_1(
         &mut self,
         elligatorswift_theirs_ephemeral_serialized: [u8; ELLSWIFT_ENCODING_SIZE],
+    ) -> Result<([u8; INITIATOR_EXPECTED_HANDSHAKE_MESSAGE_SIZE], NoiseCodec), aes_gcm::Error> {
+        let now = std::time::SystemTime::now()
+            .duration_since(std::time::UNIX_EPOCH)
+            .unwrap()
+            .as_secs() as u32;
+
+        self.step_1_with_now_rng(
+            elligatorswift_theirs_ephemeral_serialized,
+            now,
+            &mut rand::thread_rng(),
+        )
+    }
+    pub fn step_1_with_now_rng<R: rand::Rng + rand::CryptoRng>(
```

Comment:
making this method #[inline] here, will help to reduce the stack space.

Reply:
done

Reply:
done